compressors/fano_coder.py

```diff
@@ -0,0 +1,160 @@
+"""Fano Coding
+Shannon coding is a basic prefix free coding. There is some confusion in literature about Shannon, Fano and
+Shannon-Fano codes, e.g. see Naming section of https://en.wikipedia.org/wiki/Shannon–Fano_coding.
```

Comment:
somewhat paranoid - but maybe better to put the permalink to wikipedia page - left pane of wikipedia page has a link